CHANGES.md

```diff
@@ -1,3 +1,21 @@
+## v0.1.2
+
+Conduit 0.1.2 is focused on usability and debuggability.
```

Comment:
"Conduit 0.1.2 continues down the path of increasing usability and improving debugging and introspection of the service mesh itself."